### PR TITLE
Install/allinson flex

### LINKED ISSUE #80: Test allinson_flex gem
Get the IU/Notch8 application up and running in a sandbox so that the allinson_flex gem and other features can be tested.

The PR/branch is located at  https://github.com/notch8/essi/pull/48

~~Add the https://github.com/samvera-labs/allinson_flex gem to uc_drc and make the branch available for James to test.~~

This appears to require special work types.  Instructions for generating them are in the README.  We will need just one work type called "Item" for now.

Any problems with the gem should be opened as issues on the allinson_flex repo.  We can also submit PR fixes.

### PR DESCRIPTION
**Pull request by scherztc**
Fixes #80

Present short summary (50 characters or less)

This PR installs the Allinson_Flex machinery. We now have access to the Metadata Profiles in the Admin Dashboard.

We will need to create our metadata profile for the WorkType = Item.
Use this as an example : config/metadata_profile/hyrax.yaml

Then we may need to Create work types
$ rails generate allinson_flex:works

This branch should be enough to continue the research needed in issue #80

### CODE REVIEW

File: app/javascript/packs/hello_react.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+// Run this example by adding <%= javascript_pack_tag 'hello_react' %> to the head of your layout file,
```
Comment (crowesn) [inline app/javascript/packs/hello_react.jsx:1]: Is this file needed?

File: spec/features/create_item_spec.rb [hunk 2]
```diff
@@ -32,15 +32,13 @@
       login_as user
     end
 
-    scenario do
+    xscenario do
```
Comment (crowesn) [inline spec/features/create_item_spec.rb:35]: What's goin on here re "xscenario"

**Review by crowesn — COMMENTED** [review]
I think this is ready to go following our sandbox review. Please take a look at the couple comments I've included and I will merge.

File: spec/features/create_item_spec.rb (hunk 2 above)
Comment (scherztc) [reply spec/features/create_item_spec.rb:35]: @crowesn : I was following Notch 8 example for a basic feature test using Allinson Flex.  https://github.com/notch8/essi/blob/master/spec/features/create_image_spec.rb

This baffled me, too, but their test were passing and mine weren't.  It might be part of debug scramble, but they kept it in, too.

File: app/javascript/packs/hello_react.jsx (hunk 1 above)
Comment (scherztc) [reply app/javascript/packs/hello_react.jsx:1]: @crowesn  No.  I think it came over with the webpacker/reactjs generator.  Let me remove it and send the PR back up.

Comment (scherztc) [reply app/javascript/packs/hello_react.jsx:1]: Removed : https://github.com/uclibs/uc_drc/pull/93

**Comment by scherztc** [comment]
Closed with https://github.com/uclibs/uc_drc/pull/93
